Jenkinsfile

```diff
@@ -0,0 +1,91 @@
+pipeline {
+    agent {
+        label 'linux-large'
+        //TODO 6146 windows builds ?using label 'proxmox-windows'?
+    }
+    environment {
+        // What variables do we need here
+        projectName = 'DDF'
+    }
+    stages {
+        stage('Build') {
+            // Parallel Stage Syntax may soon be changing for declaritive Jenkinsfile(s)
+            // https://issues.jenkins-ci.org/browse/JENKINS-41334
+            steps{
+                parallel (
```

Comment:
I think the parallel part might work at a higher level so we could run linux and windows builds in parallel